README.md

```diff
@@ -83,6 +57,22 @@ Add the `help` flag on any command to see how you can use it. For example, `yarn
 
 The `yarn run` command will list all of the scripts available to run for this project.
 
+### Connect with server side locally
+
+After starting project up locally, you should type the following command in your browser console
+
+```
+localStorage.setItem("localdev","PUBLIC_SERVER_API")
```

Comment:
I would not bother to allow user to specify the `PUBLIC_SERVER_API`. In practice, I assume most of us will use the 9095 as the public port. Just make localdev as a boolean value.
If there is a requirement, then we can add a new property to specify that.

Also in this pr, I don't see the default for PUBLIC_SERVER_API